DatadogRUM/Sources/RUMMonitor/Scopes/RUMApplicationScope.swift

```diff
@@ -43,11 +43,6 @@ internal class RUMApplicationScope: RUMScope, RUMContextProvider {
             activeViewName: nil,
             activeUserActionID: nil
         )
-
```

Comment:
/question I know that's how Android did it, but could it be useful to not delete these, and just add the new logs?

Reply:
The logs in the view scope will be sufficient for S8S to correlate with the session, no need to keep other prints 👍